config.ini

```diff
@@ -4,13 +4,21 @@ database = database.sqlite
 [connection]
 useragent = 
 
+# Reddit backend
```

Comment:
```suggestion
```
Just commenting, but I don't think this line is needed, because of the `[reddit]` directly below it.